lib/merchant.rb

```diff
@@ -14,18 +15,13 @@ def initialize(data, repo)
   end
 
   def update(attribute)
-    instance_var_to_string
-    if @id_1 == attribute.keys[0].to_s
-      @id = attribute.values[0]
-    elsif @name_1 == attribute.keys[0].to_s
-      @name = attribute.values[0]
+    if attribute[:id] == nil
+      @updated_at = Time.now
```

Comment:
So this is essentially the way of saying the time will always update?

Reply:
this is saying that when the update method is called the @updated_at instance variable will = the time right when its updated